test/hotspot/jtreg/runtime/appcds/MoveJDKTest.java

```diff
@@ -152,12 +184,12 @@ static ProcessBuilder makeBuilder(String... args) throws Exception {
     }
 
     private static String copyFakeModulesFromHelloJar() throws Exception {
-        String classDir = System.getProperty("test.classes");
+        String outDir = CDSTestUtils.getOutputDir();
         String newFile = "hello.modules";
-        String path = classDir + File.separator + newFile;
+        String path = outDir + File.separator + newFile;
 
-        Files.copy(Paths.get(classDir, "hello.jar"),
-            Paths.get(classDir, newFile),
+        Files.copy(Paths.get(outDir, "hello.jar"),
+            Paths.get(outDir, newFile),
             StandardCopyOption.REPLACE_EXISTING);
```

Comment:
Includes the changes from [JDK-8251213](https://bugs.openjdk.org/browse/JDK-8251213) ( commit [c1093dc](https://github.com/openjdk/jdk/commit/c1093dc2447c39c0bd1e1db3627dd353662a5040) ) on this file